flow-report/assets/styles.css

```diff
@@ -0,0 +1,145 @@
+.App {
+  display: grid;
+  grid-template-areas: "side report";
+  grid-template-columns: min-content auto;
+  min-height: 100vh;
+  max-width: 100vw;
+}
+.App a {
+  color: var(--color-gray-800);
+  text-decoration: none;
+}
+
+.Separator {
+  height: 1px;
+  width: 100%;
+  background-color: var(--color-gray-300);
+  margin-top: 12px;
+  margin-bottom: 12px;
+}
+
+.Sidebar {
+  border-right: 1px solid var(--color-gray-300);
+}
+
+.Sidebar--current {
+  background-color: var(--color-gray-200);
+  font-weight: bold;
+}
+
+.SidebarSectionTitle {
+  height: 16px;
+  font-size: 11px;
+  margin: 32px 16px 0px 16px;
+  color: var(--color-gray-700);
+}
+
+.SidebarHeader {
+  margin: 16px;
+}
+.SidebarHeader__title {
+  font-size: 16px;
+  font-weight: bold;
+  width: max-content;
+}
+.SidebarHeader__date {
+  font-size: 12px;
+  color: var(--color-gray-700);
+  width: max-content;
+}
+
+.SidebarRuntimeSettings > summary {
+  padding: 4px 16px;
+  list-style: none;
+  color: var(--color-gray-700);
+  font-size: 12px;
+}
+
+.SidebarSummary {
+  width: 100%;
+  display: flex;
+  padding: 8px 16px;
+  column-gap: 16px;
+  align-items: center;
+}
+
+.SidebarSummary__icon {
+  display: flex;
+  height: 16px;
+  width: 16px;
+  align-items: center;
+  justify-content: center;
+}
+.SidebarSummary__icon::before {
+  content: '☰';
+  font-weight: normal;
+  color: var(--color-blue-A700);
+}
+
+.SidebarSummary__label {
+  font-size: 14px;
+}
+
+.SidebarFlowStep {
+  display: flex;
+  column-gap: 16px;
```

Comment:
we're on slightly shakier ground here aren't we? my safari doesn't support column-gap yet for example

Reply:
Hmm seems like it should be supported: https://caniuse.com/?search=column-gap

Reply:
In 14.1 though, 40% of Safari users won't have column gap due to the lack of evergreen enforcement. I'm not sure we ever officially established our browser support story, but this one seemed easy enough to workaround I'd bring it up.

Reply:
Aight workaround seems to be adding a margin to one of the child elements.